cypress/integration/pages/articles/tests.js

```diff
@@ -26,10 +26,56 @@ const serviceHasFigure = service =>
 const serviceHasCaption = service => service === 'news';
 // TODO: Remove after https://github.com/bbc/simorgh/issues/2962
 const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
+
 const serviceHasTimestamp = service => service === 'news';
 
-const tests = ({ service }) =>
-  describe(`Tests`, () => {
+// For testing important features that differ between services, e.g. Timestamps.
+// Inline conditional logic limiting tests to only those service/s which are different are encouraged.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`Running testsToAlwaysRun for ${service} ${pageType}`, () => {
+    if (serviceHasTimestamp(service)) {
+      it('should render a formatted timestamp', () => {
+        cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+          ({ body }) => {
+            const { language } = body.metadata.passport;
+            const { lastPublished } = body.metadata;
+            const { firstPublished } = body.metadata;
+            const updatedTimestamp = moment
+              .tz(lastPublished, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+            const firstTimestamp = moment
+              .tz(firstPublished, `${appConfig[service].timezone}`)
+              .locale(language)
+              .format('D MMMM YYYY');
+            // exempt pashto && arabic as we do have currently their locale implementation
+            if (!['pashto', 'arabic'].includes(service)) {
+              cy.get('time').then($time => {
+                if (lastPublished === firstPublished) {
+                  cy.get($time).should('contain', updatedTimestamp);
+                } else {
+                  cy.get($time)
+                    .eq(0)
+                    .should('contain', firstTimestamp);
+                  cy.get($time)
+                    .eq(1)
+                    .should(
+                      'contain',
+                      `${appConfig[service].articleTimestampPrefix}${updatedTimestamp}`,
+                    );
+                }
+              });
+            }
+          },
+        );
+      });
+    }
+  });
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
```

Comment:
`e.g. translated strings.`
There are lots of examples of translated strings (going up to two dozen)- do we want to recommend asserting these values for every single service?

Reply:
*correction: noting that we currently have 32 pages with `smoke: true` - are we okay with recommending 32 x a couple dozen tests for translations?

Reply:
These tests are actually really quick so yes, also I don't think we have 32 after this PR

Reply:
Noted. We have 7 pages now.